javascript/lifecycle.js

```diff
@@ -1,4 +1,18 @@
-import { camelize } from 'inflected'
+function camelize (str, uppercase_first_letter = true) {
```

Comment:
I'm actually changing all function definitions to use function expressions so they're block scoped and a little more strict. Also since were in JavaScript, let's use `camelCase` for the argument. I think I may also prefer having guard clause on this one.

I'd prefer to see this defined like so.

```js
const camelize = (value, uppercaseFirstLetter = true) => {
  if (typeof value !== 'string') return ''
```

Also, let's use arrow functions in the internals.

Reply:
This should now be resolved.